### DISCUSSION TITLE
Venture Capital is a Pyramid Scheme

### DISCUSSION THREAD

**Discussion by giscus** [Andrew-Chen-Wang/blog#1] [General]
# Venture Capital is a Pyramid Scheme

The VC Hypothesis is a Lie

https://blog.andrewcwang.com/blog/2024-06-30-venture-capital-is-a-pyramid-scheme/

**Comment by Andrew-Chen-Wang** [comment]
As a note, there are some public venture backed companies like Pagerduty that haven't crashed -- yet. They're primarily owned by institutional investors, still, so I'd wait and see if they turn a profit, especially as originally-narrow-focused B2B SaaSes are trumped over by platform approached SaaS or incumbents (e.g. Datadog in this case).

**Reply by Andrew-Chen-Wang** [reply]
As a note, Okta and Twilio are both not profitable by GAAP standards, and they've been around since 2009 and 2008 respectively. Okta is THE identity management standard; many companies rely on them for everything they do. Will they crash? Probably not soon, but I'm sure the CEO will leave in 5 years if they need to raise capital via equity.

These CEOs are in it to cash out. The SPAC era for these companies was a gold rush; startups weren't screwed. The employees and customers were (in terms of losing a job and having a software they rely on being lost), but never the investors or CEO.

**Comment by Andrew-Chen-Wang** [comment]
Comment from Adam: weak logic, not evidence (not really possible as VCs are private, funds of funds are private, school funds share aggregates, institutional investors don't share numbers, etc.)

conclusively: not persuasive enough from a logical standpoint given what I can provide

**Reply by Andrew-Chen-Wang** [reply]
I could potentially add SPAC mania. Good examples are FiscalNote, 23&Me, Expensify, Nextdoor, Vroom (clear example of a Ponzi scheme during the craze), several D2C brands (Love CNBC and Modern MBA's take on D2C brands and why they fail or needed to have pivoted to brick and mortar early like Warby Parker).

**Comment by Andrew-Chen-Wang** [comment]
Some things to add that I think is quite interesting; there are two types of VCs: those that inject capital and those that provide customers. I know a few hundred-million dollar VCs that are mostly ex-bankers. Without executive experience, they can't help you get enterprise customers. If you're 22, aim for a16z and Sequoia If you're 35, assuming you have a good network to sell to enterprise customers already, aim for VCs that can't provide value and only cash.

Your school matters a ton, especially when getting enterprise customers. Your alumni network can help achieve a lot so long as your school's alumni are in positions of power. Because of how the way the world works, lots of top tier schools end up becoming executives whether that be through PE or consulting. The network matters too; some alumni don't want to help their fellow alumnus.

Just interesting thoughts I had.
